CardanoSharp.Wallet/Models/Transactions/Asset.cs

```diff
@@ -0,0 +1,10 @@
+﻿#nullable enable
+namespace CardanoSharp.Wallet.Models.Transactions
+{
+    public class Asset
+    {
+        public byte[]? PolicyId { get; set; }
```

Comment:
It would be easier for consumers if we have strings instead of `byte[]`

Reply:
this would also hold `lovelaces` which this asset wouldn't have a policy id

Reply:
I'm fine with the `string` approach instead of `byte[]` 

Also this `Asset` object is used for aggregating the `Utxo` object which does mirror Koios. So you feed in your Koios Utxo object and it is flattened to the basic Asset object.

Reply:
I can move the `Asset` and the extension method used by CIP2 into the CIP2 namespace if that helps.